opentelemetry/src/main/scala/zio/telemetry/opentelemetry/tracing/Tracing.scala

```diff
@@ -685,17 +691,22 @@ object Tracing {
             getCurrentSpan.map(_.setAttribute(AttributeKey.doubleArrayKey(name), v))
           }
 
-          private def setErrorStatus[E](
+          override def setErrorStatus[E](
             span: Span,
             cause: Cause[E],
-            toErrorStatus: ErrorMapper[E]
+            errorMapper: ErrorMapper[E]
```

Comment:
I like this name. Could you please change every occurrence of `toErrorStatus` to `errorMapper` in the code base if you have time?

Reply:
changed